### PR TITLE
Include job id into progress report

### PR DESCRIPTION
**Pull request by ssyssy**
### What changes are proposed in this pull request?

Include job id into progress report

### Why are the changes needed?

better track errors.

### Does this PR introduce any user facing changes?

na

### CODE REVIEW

**Review by jja725 — COMMENTED** [review]
leave one minor request

File: dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java [hunk 1]
```diff
@@ -215,13 +215,15 @@ public void setVerificationEnabled(boolean enableVerification) {
    */
   @Override
   public void failJob(AlluxioRuntimeException reason) {
+    LOG.info("Move Job {} fails", mJobId);
```
Comment (jja725) [inline dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java:218]: do you mind put more information here like the whole job status

File: dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java [hunk 2]
```diff
@@ -215,13 +215,15 @@ public void setVerificationEnabled(boolean enableVerification) {
...
     setJobState(JobState.FAILED, true);
     mFailedReason = Optional.of(reason);
     JOB_MOVE_FAIL.inc();
   }
 
   @Override
   public void setJobSuccess() {
+    LOG.info("Move Job {} succeeds", mJobId);
```
Comment (jja725) [inline dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java:226]: here as well

File: dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java (hunk 1 above)
Comment (ssyssy) [reply dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java]: I put the job description here.

File: dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java [hunk 3]
```diff
@@ -215,13 +215,15 @@ public void setVerificationEnabled(boolean enableVerification) {
    */
   @Override
   public void failJob(AlluxioRuntimeException reason) {
+    LOG.info("Move Job {} fails with id {}", getDescription(), mJobId);
```
Comment (jja725) [inline dora/core/server/master/src/main/java/alluxio/master/job/MoveJob.java:218]:
```suggestion
    LOG.info("Move Job {} fails with status: {}", mJobId, this);
```

**Review by jja725 — CHANGES_REQUESTED** [review]

**Review by jja725 — APPROVED** [review]

**Comment by ssyssy** [comment]
alluxio-bot, merge this please

**Pull request merged**